docs/rest/Data/BundleTransaction.json

```diff
@@ -0,0 +1,76 @@
+{
+    "resourceType": "Bundle",
+    "id": "bundle-transaction",
+    "meta": {
+        "lastUpdated": "2014-08-18T01:43:30Z"
+    },
+    "type": "transaction",
+    "entry": [
+        {
+            "fullUrl": "urn:uuid:61ebe359-bfdc-4613-8bf2-c5e300945f0a",
+            "resource": {
+                "resourceType": "Patient",
+                "text": {
+                    "status": "generated",
+                    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Some narrative</div>"
+                },
+                "active": true,
+                "name": [
+                    {
+                        "use": "official",
+                        "family": "Chalmers",
+                        "given": [
+                            "Peter",
+                            "James"
+                        ]
+                    }
+                ],
+                "gender": "male",
+                "birthDate": "1974-12-25"
+            },
+            "request": {
```

Comment:
I remember a while back, @LTA-Thinking and I were discussing about only using PUT's in tests to avoid overloading databases. Something we can discuss.

Reply:
I agree. I changed two sample json files I have added to use PUT method.

Reply:
Thank you, a lot! All approved! ❤️